README.md

```diff
@@ -173,6 +173,58 @@ More will be added as they are made available
 | `log.NewDocker(...)`       | `docker-log`   | Copies logs from a docker container, via the `docker logs` command.                                                                                                                    | `container = <string,required>` <br/> `since = <duration,optional>` | 
 | `log.NewJournald(...)`     | `journald-log` | Copies logs from a journald service, via the `journalctl` command.                                                                                                                     | `service = <string,required>` <br/> `since = <duration,optional>`   |
 
+## Redactions
+
+Beginning with version `0.4.0`, `hcdiag` supports redactions. Redactions enable users to tell `hcdiag` about patterns of
+text that should be omitted from the results bundle. Redaction matching patterns are based around Regular Expressions.
+There are many implementations of Regular Expressions, but `hcdiag` uses the capabilities provided by Golang. Go's
+RegEx syntax is the same as the one documented at [https://github.com/google/re2/wiki/Syntax](https://github.com/google/re2/wiki/Syntax),
+with the exception that `\C` is not supported.
+
+Redactions are specified within custom configuration files, as shown below. Comments are included for further explanation
+of how these redactions work.
+
+```hcl
+agent {
+  # Redactions in the `agent` block will apply to all products & runners.
+  redact "regex" {
+    match = "MyPassword"
+    # This pattern will match the literal string "MyPassword". Please note that this is still a RegEx, however, so take
+    # care to properly escape any special characters that could also be used as RegEx patterns.
+  }
+}
+
+product "consul" {
+  # Redactions in the `product` block will apply to all runners within the product.
+  redact "regex" {
+    match = "127\\.0\\.0\\.1"
+    # This pattern will match the literal string "127.0.0.1"; note that RegEx requires escaping the "." with a backslash.
+    # However, HCL requires escaping the backslash with another backslash. This is why double `\`'s are required here.
```

Comment:
I appreciate these examples that help people navigate the fact that the redactions they add have to make it through two separate interpreters.